File: TEST/DAY_30_1N_ALERTDIALOG/app/src/main/java/com/example/velic/day_30_1_alertdialog/MyDialogFragment3.java
package com.example.velic.day_30_1_alertdialog;

import android.app.AlertDialog;
import android.app.Dialog;
import android.content.DialogInterface;
import android.os.Bundle;
import android.widget.Toast;

import androidx.annotation.NonNull;
import androidx.fragment.app.DialogFragment;

public class MyDialogFragment3 extends DialogFragment {
    @NonNull
    @Override
    public Dialog onCreateDialog(Bundle savedInstanceState) {

        AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
        builder.setMessage("Выберите правильный ответ")
                .setCancelable(true)
                .setPositiveButton("Мяу",
                        new DialogInterface.OnClickListener() {
                            public void onClick(DialogInterface dialog,
                                                int id) {
                                //dialog.cancel();
                                Toast.makeText(getActivity(), "Вы нажали на кнопку мяу",
                                        Toast.LENGTH_LONG).show();
                            }
                        })
                .setNeutralButton("Гав",
                        new DialogInterface.OnClickListener() {
                            public void onClick(DialogInterface dialog,
                                                int id) {
                                //dialog.cancel();
                                Toast.makeText(getActivity(), "Вы нажали на кнопку гав",
                                        Toast.LENGTH_LONG).show();
                            }
                        })
                .setNegativeButton("Сам дурак!",
                        new DialogInterface.OnClickListener() {
                            public void onClick(DialogInterface dialog,
                                                int id) {
                                //dialog.cancel();
                                Toast.makeText(getActivity(), "Вы нажали на кнопку дурак",
                                        Toast.LENGTH_LONG).show();
                            }
                        });

        return builder.create();
    }
}

File: TEST/DAY_30_1_ALERT_Dialog_kt/app/src/main/java/com/example/day_30_1_alert_dialog_k/MyDialogFragment3.kt
package com.example.day_30_1_alert_dialog_k

import android.app.AlertDialog
import android.app.Dialog
import android.content.DialogInterface
import android.os.Bundle
import android.widget.Toast
import androidx.fragment.app.DialogFragment
// Три кнопки
class MyDialogFragment3 : DialogFragment() {
    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {

        val builder = AlertDialog.Builder(activity)
        builder.setMessage("Выберите правильный ответ")
                .setCancelable(true)
                .setPositiveButton("Мяу"
                ) { dialog, id ->
                    //dialog.cancel();
                    Toast.makeText(activity, "Вы нажали на кнопку мяу",
                            Toast.LENGTH_LONG).show()
                }
                .setNeutralButton("Гав"
                ) { dialog, id ->
                    //dialog.cancel();
                    Toast.makeText(activity, "Вы нажали на кнопку гав",
                            Toast.LENGTH_LONG).show()
                }
                .setNegativeButton("Сам дурак!"
                ) { dialog, id ->
                    //dialog.cancel();
                    Toast.makeText(activity, "Вы нажали на кнопку дурак",
                            Toast.LENGTH_LONG).show()
                }

        return builder.create()
    }
}
